src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java

```diff
@@ -0,0 +1,89 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.virologischerbefund;
+
+import com.nedap.archie.rm.datavalues.DvIdentifier;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytZugehoerigeLaborprobeChoice;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytZugehoerigeLaborprobeDvUri;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytZugehoerigeLaborprobeDvIdentifier;
+
+import org.hl7.fhir.r4.model.Observation;
+
+import java.util.Optional;
+
+public class ProAnalytZugehoerigeLaborprobeChoiceConverter {
+
+    public ProAnalytZugehoerigeLaborprobeChoice convertDvIdentifier(Observation observation){
+
+        ProAnalytZugehoerigeLaborprobeDvIdentifier proAnalytZugehoerigeLaborprobeDvIdentifier = new ProAnalytZugehoerigeLaborprobeDvIdentifier();
+
+        mapDvIdentifier(observation).ifPresent(proAnalytZugehoerigeLaborprobeDvIdentifier::setZugehoerigeLaborprobe);
+
+        return proAnalytZugehoerigeLaborprobeDvIdentifier;
+
+    }
+
+    private Optional<DvIdentifier> mapDvIdentifier(Observation observation){
```

Comment:
Add this to DvIdentifierParser as mentioned in the last review.

Reply:
I moved all my parsing methods to DvIdentifierParser.java. Sadly I cannot simply modify and use the existing ones as there an Identifier is expected as an Input and I have an Observation. My main method is therefore now called "parseObservationIntoDvIdentifier" to match the old "parseIdentifierIntoDvIdentifier" in naming style.

DvIdentifierParser.java:
https://github.com/ehrbase/fhir-bridge/pull/417/files#diff-2394ecdc3d20fc7396844daa66a5a87c35e9f5a2386f87e988fb1cef9831a7b0

ProAnalytZugehoerigeLaborprobeChoiceConverter.java:
https://github.com/ehrbase/fhir-bridge/pull/417/files#diff-585a1cf0f329e2ab64afa38a03d4bfe341376a4b2b2937848f1326f68da104f9

Reply:
True one should use reflections here, but just leave it be.